test/controllers/admin/articles_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class Admin::ArticlesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
```

Comment:
remember to write tests

ALSO, this is important, I don't participate in your project but I suggest you to do a pair programing and finish some tasks together

1) finish admin namespace
2) finish at least ONE CRUD, make it ideal, nice code, specs, good UI
3) implement base CSS style guide for the app, with main CSS files, header, nav, content, forms, etc... Basically creating something what later you can re-use.

WHY?
Because you have many PR's with the same bugs and problems. First FINISH what you have right now, together.

@viktoriia-horobets @mmrshk @Bilanuk @Pakholchuk-Oleh ... tell to others too